src/js/components/DeploymentsListComponent.jsx

```diff
@@ -184,6 +191,11 @@ var DeploymentListComponent = React.createClass({
               Error fetching deployments. Refresh to try again.
             </td>
           </tr>
+          <tr className={unauthorizedClassSet}>
+            <td className="text-center text-danger" colSpan="6">
+              Error fetching deployments. Unauthorized access.
```

Comment:
Same as above – as these two `<tr>`s ar mutually exclusive I really think we could reuse the same and just change message.